Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.5'
+ruby '2.6.3'
```

Comment:
@gunjanmimo each of the other Jekyll sites that we have are on 2.6.5. Could we move the meshery.io site forward to 2.6.5 as well?